examples/fhir/lab_order.json

```diff
@@ -22,6 +31,23 @@
         }
       }
     },
+    {
+      "fullUrl": "Provenance/cd2e9813-d9cd-4cb9-bf05-7dcdb6b5fe10",
+      "resource": {
+        "resourceType": "Provenance",
+        "id": "cd2e9813-d9cd-4cb9-bf05-7dcdb6b5fe10",
+        "recorded": "2023-03-02T21:30:42.888+00:00",
+        "activity": {
+          "coding": [
+            {
+              "system": "http://terminology.hl7.org/CodeSystem/v2-0003",
+              "code": "O21",
+              "display": "OML - Laboratory order"
```

Comment:
We can, and initially I DID update the filters to pull from there. However, since ReportStream requires the Provenance resource for routing, and this info is part of it per the spec, I thought we should add it? Other examples in RS I've looked at include this info in both places, I think that's just the nature of the FHIR spec? 

If you prefer, we can remove this and update the RS filters on the receivers to look at the other place.

Reply:
Nah, no need to change it back. We can add `Provenance` to our actual code.